jsonschema-validation.xml

```diff
@@ -1043,8 +1043,16 @@
                     is going to be removed in the future.
                 </t>
                 <t>
-                    A root schema containing "deprecated" with a value of true indicates the entire
-                    root schema MAY be removed in the future.
+                    A root schema containing "deprecated" with a value of true indicates that
+                    the entire resource beng described MAY be removed in the future.
+                </t>
+                <t>
+                    When the "deprecated" keyword is used to describe an array, it can be used
+                    in the tuple form to deprecate positional schemas. If "deprecated" is used
```

Comment:
In general this is a great improvement.  Some nitpicks over language that I realized I used with you on slack but which is not really standardized so I was probably unclear.  Actually, I'm not even 100% clear on what I meant by them now that I try to write it out:

* "tuple form" means that the value of `items` is an array, and I guess I also meant that `additionalItems` is... hmm.. `false`?  Absent?  Technically a tuple would be a fixed-length array so I guess it should be `false`?  But tbh I never thought  this part through
* I don't know what I meant by "list form", if that was a term I used.  The single schema form (`items` is a single schema, `additionalItems` is irrelevant and therefore should be absent) describes a list of uniform type

Instead of "tuple form" / "list form" let's talk about "items" with an array of schemas value (and possibly with "additionalItems" also used) or "items" with a single schema value (in which case "additionalItems" is irrelevant and need not be discussed).

So if "additionalItems" is present, you can talk about deprecating individual positional schemas and/or deprecating the schema for positions after the last positionally-defined item.

I'm unsure what it ought to mean to set `"deprecated": true` in either the single schema form of `items` or in `additionalProperties`- I'd probably set `deprecated` in the schema containing those.  I think these cases are weird enough that we can ignore them for now and see if we get feedback.  It's the list-of-schemas form of `items` that is the interesting one to me.

Reply:
I agree the language should use the same form as for the keywords being reference, such as in core...

> If "items" is an array of schemas...

I'll see if I can make some suggested changes soon.